src/Loading/sceneLoader.ts

```diff
@@ -976,4 +978,143 @@ export class SceneLoader {
             }, pluginExtension);
         });
     }
+
+    /**
+     * Import animations from a file into a scene
+     * @param rootUrl a string that defines the root url for the scene and resources or the concatenation of rootURL and filename (e.g. http://example.com/test.glb)
+     * @param sceneFilename a string that defines the name of the scene file or starts with "data:" following by the stringified version of the scene or a File object (default: empty string)
+     * @param scene is the instance of BABYLON.Scene to append to (default: last created scene)
+     * @param targetConverter defines a function used to convert animation targets from loaded scene to current scene (default: search node by name)
+     * @param onSuccess a callback with the scene when import succeeds
+     * @param onProgress a callback with a progress event for each file being loaded
+     * @param onError a callback with the scene, a message, and possibly an exception when import fails
+     */
+    public static ImportAnimations(rootUrl: string, sceneFilename: string | File = "", scene: Nullable<Scene> = EngineStore.LastCreatedScene, targetConverter: Nullable<(target: any) => any> = null, onSuccess: Nullable<(scene: Scene) => void> = null, onProgress: Nullable<(event: SceneLoaderProgressEvent) => void> = null, onError: Nullable<(scene: Scene, message: string, exception?: any) => void> = null): void {
+        if (!scene) {
+            Logger.Error("No scene available to load animations to");
+            return;
+        }
+
+        let _targetConverter = targetConverter ? targetConverter : this._defaultTargetConverter(scene);
+
+        let onAssetContainerLoaded = (container: AssetContainer) => {
+            SceneLoader.MergeAnimations(scene, container, _targetConverter);
+
+            container.dispose();
+
+            if (onSuccess) {
+                onSuccess(scene);
+            }
+        };
+
+        // Reset, stop and dispose all animations before loading new ones
+        for (let animatable of scene.animatables) {
+            animatable.reset();
+        }
+        scene.stopAllAnimations();
+        scene.animationGroups.slice().forEach(animationGroup => {
+            // animationGroup.dispose();
+        });
+        let animatableObjects = this._getAllAnimatableObjects(scene);
+        animatableObjects.forEach(animatableObject => {
+            animatableObject.animations = new Array<Animation>();

```

Comment:
Not sure to get why we need this

Reply:
Simply a clean up of the mesh.animations, light.animations, etc when importing in overwrite mode.
The append mode will not do that of course.

Reply:
I assume we overwrite animations with same target property as the ones being loaded, even in append mode to get a clean array, right ?
Anyway, the animation classes are already handling the case where a target node has several animations with same target property.

Reply:
> I assume we overwrite animations with same target property as the ones being loaded, even in append mode to get a clean array, right ?

Correct

Reply:
Done in https://github.com/BabylonJS/Babylon.js/pull/7252/commits/bd48adfd1f20ddeec757a42990793adff611fd48